### PR TITLE
keystone deployment fixes

### PR DESCRIPTION
**Pull request by krehermann**
[KS-486](https://smartcontract-it.atlassian.net/browse/KS-486): fix fowarder signer address
[KS-487](https://smartcontract-it.atlassian.net/browse/KS-487) add support for aptos keys in ocr3 contract config
[KS-488](https://smartcontract-it.atlassian.net/browse/KS-488): support ocr3 contract config updates
and add more logging 
### Resolves

[KS-486]: https://smartcontract-it.atlassian.net/browse/KS-486?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[KS-487]: https://smartcontract-it.atlassian.net/browse/KS-487?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[KS-488]: https://smartcontract-it.atlassian.net/browse/KS-488?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: integration-tests/deployment/keystone/changeset/deploy_ocr3.go [hunk 1]
```diff
@@ -38,3 +39,29 @@ func DeployOCR3(lggr logger.Logger, env deployment.Environment, ab deployment.Ad
 	return deployment.ChangesetOutput{AddressBook: ab}, nil
 
 }
+
+func ConfigureOCR3Contract(lggr logger.Logger, env deployment.Environment, ab deployment.AddressBook, registryChainSel uint64, nodes []*models.Node, cfg kslib.OracleConfigSource) (deployment.ChangesetOutput, error) {
+	// must have capabilities registry deployed
```
Comment (bolekk) [inline integration-tests/deployment/keystone/changeset/deploy_ocr3.go]: why?

File: integration-tests/deployment/keystone/deploy.go [hunk 2]
```diff
@@ -724,25 +763,47 @@ func registerDons(lggr logger.Logger, req registerDonsRequest) (*registerDonsRes
 			return nil, fmt.Errorf("failed to confirm AddDON transaction %s for don %s: %w", tx.Hash().String(), don, err)
 		}
 		lggr.Debugw("registered DON", "don", don, "p2p sorted hash", p2pSortedHash, "cgs", cfgs, "wfSupported", wfSupported, "f", f)
+		registeredDons++
+	}
+	lggr.Debugf("Registered all DONS %d, waiting for registry to update", registeredDons)
+
+	// occasionally the registry does not return the expected number of DONS, so we retry a few times
```
Comment (bolekk) [inline integration-tests/deployment/keystone/deploy.go]: does it happen only right after update? or randomly..?

File: integration-tests/deployment/keystone/types.go [hunk 3]
```diff
@@ -130,6 +161,10 @@ func newOcr2Node(id string, ccfgs map[chaintype.ChainType]*v1.ChainConfig, csaPu
 	}
 
 	var sigb [32]byte
+	// initialize sigb to 0s
```
Comment (bolekk) [inline integration-tests/deployment/keystone/types.go]: Doesn't Go do that already?

**Review by bolekk — DISMISSED** [review]

File: integration-tests/deployment/keystone/deploy.go (hunk 2 above)
Comment (krehermann) [reply integration-tests/deployment/keystone/deploy.go]: only right after

File: integration-tests/deployment/keystone/types.go (hunk 3 above)
Comment (krehermann) [reply integration-tests/deployment/keystone/types.go]: yes. this is leftover, when i was confused about the prefix we where seeing on chain, which is from the multichain encoding

File: integration-tests/deployment/keystone/changeset/deploy_ocr3.go (hunk 1 above)
Comment (krehermann) [reply integration-tests/deployment/keystone/changeset/deploy_ocr3.go]: cruft. removed

**Review by bolekk — APPROVED** [review]

**Review by vyzaldysanchez — APPROVED** [review]

**Pull request merged**